packages/rs-dpp/src/errors/consensus/codes.rs

```diff
@@ -64,6 +65,32 @@ impl ErrorWithCode for BasicError {
             Self::DataContractUniqueIndicesChangedError { .. } => 1053,
             Self::DataContractInvalidIndexDefinitionUpdateError { .. } => 1054,
             Self::DataContractHaveNewUniqueIndexError { .. } => 1055,
+            Self::InvalidDocumentTypeRequiredSecurityLevelError { .. } => 1071,
+            Self::UnknownSecurityLevelError { .. } => 1072,
+            Self::UnknownStorageKeyRequirementsError { .. } => 1073,
+            Self::ContractError(DataContractError::DecodingContractError { .. }) => 1074,
+            Self::ContractError(DataContractError::DataContractAlreadyExistsError { .. }) => 1075,
```

Comment:
Are you sure it's basic validation error?

Reply:
Removed that error